### ISSUE TITLE
Failed to copy output to clipboard: XcbConnect(DisplayParsingError)

### ISSUE THREAD

**Issue by AnirudhRahul** [mufeedvh/code2prompt#15]
Could a flag be added to skip copying the output to clipboard so code2prompt can be used on a remote without a clipboard.

**Comment by ahmedbentabet** [comment]
### Quick Solution: Remove Clipboard Functionality and Recompile

The `arboard::Clipboard` functionality in `main.rs` is causing issues. To resolve this, we need to remove the related lines of code and recompile the project.

1. **Remove the following lines from `main.rs`:**

    ```rust
    use arboard::Clipboard;
    ...
    let mut clipboard = Clipboard::new().expect("Failed to initialize clipboard");
    clipboard.set_text(rendered).expect("Failed to copy output to clipboard");

    println!(
        "{}{}{} {}",
        "[".bold().white(),
        "✓".bold().green(),
        "]".bold().white(),
        "Prompt copied to clipboard!".green()
    );
    ```

2. **Recompile the project:**

    ```sh
    cargo build --release
    ```

**Comment by ODAncona** [comment]
@AnirudhRahul I implemented the `--no-clipboard` option in #20. If you are on a remote machine, I'm able to use it using the `--output` flag.

**Comment by mufeedvh** [comment]
Fixed in #20 by @ODAncona! 🙌 🎉
